### PR TITLE
Stop using AnimatableManager in Hint

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
- Remove usage of AniamtableManger from our Hint component
- Use Colors.primary instead of Colors.blue30 

## Changelog
* Replace usage of `react-native-animatable` from our Hint component with our own implementation

### CODE REVIEW

File: src/components/hint/index.js [hunk 1]
```diff
@@ -343,19 +353,18 @@ class Hint extends BaseComponent {
...
-    const shownUp = position === HINT_POSITIONS.TOP;
+    const {message, messageStyle, icon, iconStyle, borderRadius, color, testID} = this.getThemeProps();
 
     if (this.showHint) {
       return (
-        <AnimatableView
-          animation={shownUp ? AnimatableManager.animations.hintAppearUp : AnimatableManager.animations.hintAppearDown}
-          duration={200}
+        <View
+          animated
```
Comment (Inbal-Tish) [inline src/components/hint/index.js:361]: how do you control the type of animation to apply on the View?

Comment (ethanshar) [reply src/components/hint/index.js:361]: What do you mean?

Comment (Inbal-Tish) [reply src/components/hint/index.js:361]: With Animatable you can load the animations, like 'hintApearUp'. How do you do it with the 'animated' prop of View. What animation does the Hint will have now?

Comment (Inbal-Tish) [reply src/components/hint/index.js:361]: Got it. getHintAnimatedStyle

**Review by Inbal-Tish — APPROVED** [review]

File: src/components/hint/index.js (hunk 1 above)
Comment (ethanshar) [reply src/components/hint/index.js:361]: The default one. 
Users won't be able to configure it now. 
I don't thing people were aware of that and I think it's pretty redundant..
Of course if someone will request we can reconsider it

**Pull request merged**